regression/goto-gcc/gcc_builtins4/main.c

```diff
@@ -7,6 +7,7 @@ union { int i; } u;
 enum { Econst } e;
 int a[10];
 
+#ifndef __clang__
```

Comment:
I'd appreciate if all the `#ifndef __clang__` changes could be placed in a separate commit; at the bare minimum there should be some information in the commit message.